### ISSUE TITLE
If data is all 0, then axis is NaN

### ISSUE THREAD

**Issue by silviajoy** [oksktank/react-native-pure-chart#8]
```
let graph = [0,0,0,0,0] 
render() {
    <View>
         <PureChart data={graph} type='line' />
    </View>
}
````

renders this:
![nan](https://user-images.githubusercontent.com/16961449/36587605-f2dcc506-1885-11e8-8455-12e20b05f1f0.jpg)

Am I doing something wrong or there is an issue?

Thanks :)

**Comment by oksktank** [comment]
Hi, @silviajoy 

We noticed that we have a drawing bug when all y values are zero.
I fixed it and release 0.0.16.

please checkout new version of react-native-pure-chart.

Thank you.

**Comment by silviajoy** [comment]
Thank you! :)
